packages/teams-js/src/public/tasks.ts

```diff
@@ -47,14 +48,14 @@ export namespace tasks {
       : undefined;
     if (taskInfo.card === undefined && taskInfo.url === undefined) {
       ensureInitialized(runtime, FrameContexts.content, FrameContexts.sidePanel, FrameContexts.meetingStage);
-      sendMessageToParent('tasks.startTask', [taskInfo as DialogInfo], submitHandler);
+      sendMessageToParentWithVersion('v1', 'tasks.startTask', [taskInfo as DialogInfo], submitHandler);
     } else if (taskInfo.card) {
       ensureInitialized(runtime, FrameContexts.content, FrameContexts.sidePanel, FrameContexts.meetingStage);
-      sendMessageToParent('tasks.startTask', [taskInfo as DialogInfo], submitHandler);
+      sendMessageToParentWithVersion('v1', 'tasks.startTask', [taskInfo as DialogInfo], submitHandler);
     } else if (taskInfo.completionBotId !== undefined) {
-      dialog.url.bot.open(getBotUrlDialogInfoFromTaskInfo(taskInfo), dialogSubmitHandler);
+      botUrlOpenHelper(getBotUrlDialogInfoFromTaskInfo(taskInfo), dialogSubmitHandler);
```

Comment:
This is purely from a person's point of view who doesn't interact with teamsjs regularly so this comment is mostly a personal opinions as headup :)
How do we feel to make apiVersion as required in botUrlOpenHelper (and also other helpers), meaning here we will also pass V1 instead of letting it pick up the default V1 when not specifying? The way I am correlating is: Tasks is an old capability, so anything coming out of it should be V1. If we pass v1 specially here, I don't have to go into botUrlOpenHelper to check what version being sent to it. It applies to other capabilities being specific about versioning.

on a related note, I found it is really helpful to be specific passing v1 in  Dialog.ts

Reply:
I think it makes more sense. Version number is much more important so we could tolerate a little bit inconvenience on writing code. Will make the change and force developers to specifically label the version for APIs.